esp/templates/survey/review.html

```diff
@@ -12,19 +12,68 @@
 <style type="text/css">
 .nocheckmark { border: 1px solid black; }
 </style>
-<br /><br />
-<h1>Survey Responses for {{ program.niceName }} </h1>
+<a name="top"></a><h1>Survey Responses for {{ program.niceName }} </h1>
+
+{% if teacher_form %}
+<form method="POST" action="{{ request.get_full_path }}">
+<h2>Search for a teacher to retrieve their specific survey results:</h2>
+{{ teacher_form.target_user }}
+<input type="submit" value="Search" />
```

Comment:
If I hit "Search" with no teacher entered (yes this is not the point but still), I get the following error:

![image](https://user-images.githubusercontent.com/25392075/78617732-3c201200-7846-11ea-85ff-0e956ae17951.png)